examples/kubeval/README.md

```diff
@@ -0,0 +1,47 @@
+# Kubeval
+
+The `kubeval` KRM config function validates Kubernetes resources using kubeval.
+Learn more on the [kubeval website].
+
+This example invokes the kubeval function against Kubernetes v1.18.0 using
+declarative configuration.
+
+## Function invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/kubeval .
+kpt fn run kubeval --network --results-dir /tmp
```

Comment:
Is there a e2e test for `--results-dir`?

Reply:
Added e2e tests for --results-dir and added TODOs linking to issues about removing --results-dir from examples.